log/log.go

```diff
@@ -115,8 +118,27 @@ func LevelStr(level int) (string, error) {
 
 func logTag(tag string, format string, a ...interface{}) {
 	f := fmt.Sprintf("[%s] %s\n", tag, format)
-	str := fmt.Sprintf(f, a...)
-	log.Printf(str)
+	output := fmt.Sprintf(f, a...)
+
+	if output != lineLast {
```

Comment:
@dorileo 
If you manually increase logging to Debug level, you see output like:
```
2018/10/12 14:49:26 [DBG] Warning: The resulting partition is not properly aligned for best performance.
2018/10/12 14:49:26 [DBG] [Previous line repeated 2 times]
2018/10/12 14:49:43 [DBG] Starting download of remaining update content. This may take a while...
2018/10/12 14:49:43 [DBG] .
2018/10/12 14:49:43 [DBG] [Previous line repeated 63 times]
2018/10/12 14:49:43 [DBG] Finishing download of update content...
2018/10/12 14:49:43 [DBG] Installing bundle(s) files...
2018/10/12 14:49:43 [DBG] .
2018/10/12 14:49:50 [DBG] [Previous line repeated 127 times]
2018/10/12 14:49:50 [DBG] Calling post-update helper scripts.
```

Do you think it is really useful to have a single like with a '.' logged 128 times?
If so, I'll consider adding a flag or an even more vebose logging level.

Reply:
This is not something I *personally* consider as useful, but these dots has been requested as an indication of swupd activity in the past and it may be requested again. That's the only concern, if we can flag we can still provide the "feature" if such kind of requirement shows up.